notebooks/apify_haystack_rag.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>We can give the link of the github repo instead of the PyPI package here <a href="https://github.com/apify/apify-haystack/tree/main" target="_blank">https://github.com/apify/apify-haystack/tree/main</a>.</p><p>Also, we should say "Haystack website" without AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>